### PR TITLE
Add osp17.0 fencing profile

### PR DESCRIPTION
**Pull request by udis**

### CODE REVIEW

File: ansible/vars/17.0_fencing.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+enable_fencing: true
+osp_release_auto_version: 17.0-RHEL-9
+ephemeral_heat:
```
Comment (udis) [inline ansible/vars/17.0_fencing.yaml:4]: Now that the downstream images are available how do we make use of them?

Comment (pinikomarov) [reply ansible/vars/17.0_fencing.yaml:4]: #osp17 testing. you can use
director_operator_image: quay.io/mschuppe/osp-director-operator
csv_version: 17.0.35-17.0

**Comment by pinikomarov** [comment]
@udis I reran the failed gate job : https://rhos-ci-jenkins.lab.eng.tlv2.redhat.com/blue/organizations/jenkins/DFG-ospk8s-osp-director-dev-tools-multibranch/detail/PR-439/3/pipeline

File: ansible/vars/17.0_fencing.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+#ephemeral_heat:
+#  heat_api_image: quay.io/tripleowallaby/openstack-heat-api:current-tripleo
+#  heat_engine_image: quay.io/tripleowallaby/openstack-heat-engine:current-tripleo
+#  mariadb_image: quay.io/tripleowallaby/openstack-mariadb:current-tripleo
+#  rabbit_image: quay.io/tripleowallaby/openstack-rabbitmq:current-tripleo
+director_operator_image: quay.io/mschuppe/osp-director-operator
+csv_version: 17.0.35-17.0
+
+osp_release_defaults:
+  base_image_url: http://download.devel.redhat.com/brewroot/packages/rhel-guest-image/9.0/20220428.0/images/rhel-guest-image-9.0-20220428.0.x86_64.qcow2
```
Comment (stuggi) [inline ansible/vars/17.0_fencing.yaml:14]: this image is no longer available. in https://github.com/openstack-k8s-operators/osp-director-dev-tools/pull/440 I update it to be http://download.devel.redhat.com/brewroot/packages/rhel-guest-image/9.0/20221216.0/images/rhel-guest-image-9.0-20221216.0.x86_64.qcow2

File: ansible/vars/17.0_fencing.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+enable_fencing: true
+# As OSP17 Fencing is required for d/s testing we will use the custom index instead of the parameteres below
+#osp_release_auto_version: 17.0-RHEL-9
+#ephemeral_heat:
+#  heat_api_image: quay.io/tripleowallaby/openstack-heat-api:current-tripleo
+#  heat_engine_image: quay.io/tripleowallaby/openstack-heat-engine:current-tripleo
+#  mariadb_image: quay.io/tripleowallaby/openstack-mariadb:current-tripleo
+#  rabbit_image: quay.io/tripleowallaby/openstack-rabbitmq:current-tripleo
+director_operator_image: quay.io/mschuppe/osp-director-operator
+csv_version: 17.0.35-17.0
```
Comment (stuggi) [inline ansible/vars/17.0_fencing.yaml:11]: we might want to set this in the job and leave it to the upstream repo and `csv_version: latest-17.0`

File: ansible/vars/17.0_fencing.yaml (hunk 2 above)
Comment (stuggi) [reply ansible/vars/17.0_fencing.yaml]: and update this

File: ansible/vars/17.0_fencing.yaml (hunk 3 above)
Comment (udis) [reply ansible/vars/17.0_fencing.yaml]: Updated the pr with relevant changes for osp17

**Pull request merged**